src/pages/Search/SearchPage.tsx

```diff
@@ -537,6 +538,30 @@ function SearchPage({route}: SearchPageProps) {
             });
         }
 
+        const shouldShowChangeApproverOption =
+            !isOffline &&
```

Comment:
Can't do this while offline? This could be a follow up potentially - but I can't think of a clear reason to prevent it.